main.go

```diff
@@ -71,6 +72,8 @@ func main() {
 	logs.AddFlags(cmd.Flags())
 	cmd.Flags().BoolVar(&cmd.Insecure, "insecure", false, "if true authentication and authorization are disabled, only to be used in dev mode")
 	cmd.Flags().IntVar(&cmd.MonitoringPort, "monitoring-port", 9090, "port to expose readiness and Prometheus metrics")
+	cmd.Flags().BoolVar(&cmd.EnableProfiling, "enable-profiling", false, "If true starts a http server for pprof profiling")
+	cmd.Flags().IntVar(&cmd.ProfilingPort, "profiling-port", 8085, "port to expose pprof profiling")
```

Comment:
I wonder if we want to do what we did in ECK and if the port is `0` it's disabled, otherwise it's enabled?

https://github.com/elastic/cloud-on-k8s/blob/3aa17627dcb48acf6d72c48e89e12a8c338cad47/deploy/eck-operator/values.yaml#L160-L161

Reply:
Made the changes similar to your suggestion